xml2json/lib.py

```diff
@@ -1,7 +1,6 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 import lxml.etree
-import six
```

Comment:
Was `six` installed anywhere? I don't see it in setup.py or travis which is confusing

Reply:
No, this was my first commit because I couldn't get tests to pass without installing it.  I'm not sure how the tests were passing up till now (maybe they weren't?).

Reply:
Ah, cancel that, `unittest2` depends on `six`, so it was getting installed implicitly with `python setup.py test`.

Reply:
💡 gotcha thanks that makes sense!